src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--grey-color);
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --black-color: #000;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --grey-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  min-width: 1024px;
```

Comment:
remove min and max width. In next tasks you use this header but it's should be for full width